lib/pact_broker/deployments/environment_service.rb

```diff
@@ -34,6 +33,10 @@ def self.find_by_name(name)
       def self.delete(uuid)
         PactBroker::Deployments::Environment.where(uuid: uuid).delete
       end
+
+      def self.find_for_pacticipant(pacticipant)
```

Comment:
Not using the pacticipant?

Reply:
Well spotted - that's a dummy method for Pactflow to override by supplying a different service. In the OSS version, any pacticipant can deploy to any environment (mostly because I don't want to have to develop a configuration page 😆 ), but in Pactflow, I envisage us supporting assigning particular environments to pacticipants.